### ISSUE TITLE
checkIgnore "isUncaught" argument is always `false`

### ISSUE THREAD

**Issue by highspotjeremy** [rollbar/rollbar.js#881]
The `isUncaught` (first) argument in the `checkIgnore` function call is always `false`. This change appears to be the result of PR #850. Specifically, [this line](https://github.com/rollbar/rollbar.js/pull/850/files#diff-c854b4a38519d3019f2e50f74e73bf5fR106) deletes the `_isUncaught` property on the item before the `userCheckIgnore` predicate is being called. Prior to this change, that property was deleted in that predicate method.

Code sample:
```javascript
const options = {
  checkIgnore: (isUncaught) => {
    console.log(isUncaught);
  }
};
Rollbar.configure(options);
window.onerror(new Error('test'));
```

This should produce a `true` in the console since the `window.onerror` hook is getting called, but it instead produces `false`. (I can replicate this by doing an actual `throw`.)

**Comment by waltjones** [comment]
@highspotjeremy Thank you.

**Comment by waltjones** [comment]
@highspotjeremy Fix released in v2.19.2.

**Comment by highspotjeremy** [comment]
@waltjones - thank you!
